leveldown.js

```diff
@@ -88,8 +89,8 @@ SubDown.prototype._open = function (opts, cb) {
     var subdb = reachdown(self.db, 'subleveldown')
 
     if (subdb && subdb.prefix) {
-      self.prefix = subdb.prefix + self.prefix
-      self.leveldown = reachdown(subdb.db, matchdown, false)
+      self.prefix = subdb.prefix + self.ownPrefix
+      self.leveldown = subdb.leveldown || reachdown(subdb.db, matchdown, false)
```

Comment:
We should only need 1 one of these solutions (either `subdb.leveldown` or `reachdown(..)`), not sure which I like best.

Reply:
Gonna go with `subdb.leveldown` because `reachdown(sub.db)` is like taking a detour (it goes: own levelup -> encoding-down -> subleveldown -> external levelup -> encoding-down -> leveldown).